public/js/chess.js

```diff
@@ -51,6 +51,13 @@ class ChessBoard {
     }
   }
 
+  rookIsValidMove(startCoordinate, endCoordinate){
+    var rooksMoveHorizontallyandVertically = startCoordinate.x === endCoordinate.x || startCoordinate.y === endCoordinate.y
+    if(rooksMoveHorizontallyandVertically && this.isEmpty(endCoordinate)){
```

Comment:
up to you, but i think that this would achieve the same effect and be smaller:
` return (rooksMoveHorizontallyandVertically && this.isEmpty(endCoordinate)`